charts/jindofs/templates/fuse/daemonset.yaml

```diff
@@ -107,6 +107,24 @@ spec:
               readOnly: true
           {{- end }}
       restartPolicy: Always
+      {{- if $notEnableDnsConfig }}
+      dnsPolicy: ClusterFirstWithHostNet
+      {{- else }}
+      dnsConfig:
+        nameservers:
+        - {{ .Values.master.dnsServer }}
+        options:
+        - name: ndots
+          value: "5"
+        searches:
+        - {{ .Values.master.namespace }}.svc.cluster.local
+        - svc.cluster.local
+        - cluster.local
+        - tbsite.net
+        - aliyun.com
+      dnsPolicy: None
```

Comment:
I suggest to make searches configurable. Actually, the following domain names are not common. We can create a new PR to fix it  later. thanks. @frankleaf